### DISCUSSION TITLE
On-Chain Identity Space

### DISCUSSION THREAD

**Discussion by markopoloparadox** [capsule-corp-ternoa/ternoa-hub#5] [General]
Hello guys, to speed up this process prepare all the necessary documents to prove the ownership of your address(s), and any other additional information that you have filled up. As an example: User Cryptonode - 01 needs to provide proof that he owns the entered website, twitter account and riot account.
![image](https://user-images.githubusercontent.com/4882434/175948789-e892ad1d-621e-401a-bbcc-c0199c2fe218.png)

Rules:
- One Message per account
- Any message that is not an identity proof will be deleted. If you have a questions use #7 space for that.

**Comment by WyvernNE** [comment]
# Crystal-Network Proof of Identity:

## Address:
5H5zYAUc37j9MzKhjN5drx2UMDATTcyTQ39XEAS6b6Jd7cSo

![image](https://user-images.githubusercontent.com/17429470/175957267-939e85e3-957f-469d-a604-15af70dac100.png)

## Email:
![image](https://user-images.githubusercontent.com/17429470/175955905-04242539-6bba-473c-a4fe-a0a37e13bbd6.png)

## Twitter:
![image](https://user-images.githubusercontent.com/17429470/175956398-8c08d6f3-7ce7-402c-816e-2537b4fb7ea9.png)
![image](https://user-images.githubusercontent.com/17429470/175956474-166d4af8-f780-48f3-9096-c77c50cb3137.png)

**Comment by Orius** [comment]
### Orius International proof of identity

1) Adresses
5H8miAPvJLkgVBbQpaTgq1s45CqSK5ztjtE3iJvZPo8pqaq6
5DjnVuRmNvqFmrokATt7RGEeHdwYXoq1gyaYVT1NbQyAB39S
5EtF7pEziw7xwvGZqftjRf3GH4HFGoejXqYmGzGyxWi8Yr7B
5EcAqQ8KfS2b5TtEKEe4EBeH2humTvnMdRkjqTB3NPT2Kws2
5H1mGSgTR3Gvsq1HDnWnRozQdGenJsqhVQo1xKr7PwG5x5dU

![image](https://user-images.githubusercontent.com/64799905/175982350-6749c28a-606d-45b6-b2be-eeb6d2ebf726.png)
![image](https://user-images.githubusercontent.com/64799905/175982501-91c6ef21-262e-47a2-b5a5-6ea23b0cb6c2.png)

2) Email contact@orius.bz
![image](https://user-images.githubusercontent.com/64799905/179362050-3bc78e19-4339-46c0-b783-3d42785a7836.png)

3) Twitter account @OriusLLC
![image](https://user-images.githubusercontent.com/64799905/175962779-a99b4a25-0c3a-45cd-9161-d13deec7aac2.png)

**Comment by Crypte** [comment]
### Proof of identity

Adresses : 

CRYPTENODE - 01    5ECM7nUB5VCuTBGa9V5xeCECr4CDr7d3M4gGK4DLfzCvuBBD
CRYPTENODE - 02    5EeeRBQSzx5mmtT5EjPmfEj21GbNRAGy71Q1Q3UTYSuk5Gy4
CRYPTENODE - 03    5HnLknMoTe9sLRd5bNjf9ZwYiZgtaRpgqYMjF6kSduPPxa97

![image](https://user-images.githubusercontent.com/92172988/177427413-1d46eac8-2db6-45f1-a4b9-db9af73bb870.png)

Website : https://crypte.gitbook.io

![image](https://user-images.githubusercontent.com/92172988/176051522-8a7807db-5ad8-47ec-ab71-32a2ce0332c2.png)

Twitter : https://twitter.com/_Crypte_

![image](https://user-images.githubusercontent.com/92172988/176051600-f0420c7f-28b4-487d-8f03-477a12d5ae86.png)

Riot / Matrix : @louis_crypte:matrix.org

![image](https://user-images.githubusercontent.com/92172988/176051698-bea8df0b-d2bb-4a94-bad5-13be126c2f76.png)

**Comment by Tulleb** [comment]
**Proof of identity**

Address:
- Tulleb23 - 5FXAQYymZui6nGNxCer6ShQ3jMfEdDS46aowonEYfFngJ49a

<img width="1818" alt="image" src="https://user-images.githubusercontent.com/5658984/176138624-30130232-42d3-4786-a6a8-c92119e0d8ef.png">

Email:
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/5658984/176139910-d3f33775-b156-4c6e-a139-cc9a513c4202.png">

**Comment by d4rk0snet** [comment]
Proof of identity
Adresses :

EARTH - 5EeF2smfChbJs8unGY6JMFdiXWya2ro16EjCFA9GRFPBFXa9
MARS - 5Dc6uTzgiFrMWv6Sn14uvWKKSf8dBUeLca1HZEKXQWPRyyDJ
![Capture du 2022-06-28 15-10-50](https://user-images.githubusercontent.com/92326710/176188622-1044697c-030e-43be-b0bd-4bad99dfdb09.png)

**Comment by ipapandinas** [comment]
HERACLES proof of identity

Adress
5FWuM8Q3DRBAzu2PeqfmXqwdnegk8yiiMLXtZLB3dqJjomG8

<img width="886" alt="heraclesAccount" src="https://user-images.githubusercontent.com/26460174/176201150-100c9ce9-fec1-4a4c-a853-4842222c97f5.png">

Website : https://heracles.works/

![websiteHeracles](https://user-images.githubusercontent.com/26460174/176201887-be69d117-0312-4ed8-9135-7beac9379301.png)

Twitter : https://twitter.com/heracles_works

<img width="675" alt="twitterHeracles" src="https://user-images.githubusercontent.com/26460174/176202264-569bc5ef-d56d-4f01-831d-e3c6a443d42f.png">

Mail : contact@heracles_works

![mailHeracles](https://user-images.githubusercontent.com/26460174/176202367-3b89452b-56a7-4a7b-9a94-7da70ceb4a97.png)

**Comment by t0sma86** [comment]
### TOSMA NODE : Proof of Identity 

Adress : **5CFpjyHHojmWAWoPfyoxbgwhzSf4ZELPzwU7oQtny6xVCjtL**
![Tosma NODE](https://user-images.githubusercontent.com/108297916/176249084-39936036-ed28-4a85-9571-10544965f5a3.png)

TWITTER : 
![Twitter](https://user-images.githubusercontent.com/108297916/176249178-ae12a19e-5d28-4e35-9de0-3d792212647f.png)

**DISCORD :** 
![Discord](https://user-images.githubusercontent.com/108297916/176249208-ff09ab30-c5c2-42c7-8759-fea80a74deee.png)

**ACCESS POLKADOT JS :** 
![Polkadot JS](https://user-images.githubusercontent.com/108297916/176249266-40271d0a-df1a-4b39-9123-cdae49a8f02e.png)

**ACCESS TO SERVERS T0smaNode-01 & T0smaNode-02** (running ternoa.service)
![T0smaNode-01](https://user-images.githubusercontent.com/108297916/176249393-9fd0a524-1c37-4fcc-ba67-665ab2eaeb0c.png)
![T0smaNode-02](https://user-images.githubusercontent.com/108297916/176249686-b3d7a238-89e9-4b0c-a7f7-e2c7d42b5d3b.png)

**Reply by t0sma86** [reply]
Hello, For confirmation of my 2nd node : 5EyFqdBXjyUXHEk3Thuxy5tVD617nyD8xnUYSAUY7DvoeMdJ

SERVEUR : Call 03 on telemetry but it's 02 for staking 
![serveur](https://user-images.githubusercontent.com/108297916/177477030-6065996c-ca4f-4750-9d9c-67dfddbaba33.png)

Polkadot JS

![polka](https://user-images.githubusercontent.com/108297916/177477063-8f96ccff-f6a7-4d6d-ab18-a9227a1230f9.png)

**Reply by t0sma86** [reply]
Hello, For confirmation of my  3rd node : 5C8MZwoVMSRf8nBezfWvNigix9PhUPEiDnjbZcixUFcwh5mG

Polkadot JS : 

![Capture d’écran 2022-07-30 233848](https://user-images.githubusercontent.com/108297916/181997068-33ecded0-f52c-478a-9ee4-4380d6b7e964.png)

Serveur with Ternoa Service 👍 
![Capture d’écran 2022-07-30 234001](https://user-images.githubusercontent.com/108297916/181997071-50c8b502-7167-40dd-b1f8-87f998112733.png)

**Comment by cryptokyser** [comment]
**CK-VALIDATOR - Proof of Identity**

Hello team, below, proof of my Idendity.

Adress : 5HYnwpmmvFc6Ub5SkoLKCTdgB8conmMS18JmJocBqUKG1Bf7

![image](https://user-images.githubusercontent.com/109348936/179205015-75463c4b-4275-447a-a4fc-a18716052073.png)


Twitter : https://twitter.com/CryptoKyser


![image](https://user-images.githubusercontent.com/109348936/179205332-e6b5e305-cc7a-4c83-8d3a-6893b9d5019c.png)


Mail : contact@cryptokyser.com


![image](https://user-images.githubusercontent.com/109348936/179205630-948dee1c-a548-41a1-98a7-f73602f95b3a.png)

Regards,

CryptoKyser

**Comment by Orius** [comment]
Hello team.
The email address of Orius International has changed.
The proof has been updated in the original post above ☝
The "good know" was removed when I implemented this change.
Can you be so kind as to restore them, on the 5 addresses concerned?
Thanks 🙏

**Comment by MNPL-Monopole** [comment]
### MONOPOLE 01 proof of identity

**1-Node Adress**
5F9ad4MXrzDHJ4kmCSJxvG27L6a2SkTtNhnaMJE52WRyc1Q9

![image](https://user-images.githubusercontent.com/111024782/184039030-41f28a78-b38f-4407-8550-4fb0b7627581.png)

**2-Email founders@monopole.email**

![image](https://user-images.githubusercontent.com/111024782/184039150-e968a982-6b45-4373-82d3-777dcc1dcbec.png)

**3-Website https://linktr.ee/mnpl**

![image](https://user-images.githubusercontent.com/111024782/184039358-917f1b21-babd-4053-908b-f302b8d2a0a7.png)

4-Twitter @monopole_mnpl

![image](https://user-images.githubusercontent.com/111024782/184039494-9b354257-13a4-4f5a-b2b5-a2523a0e5bde.png)

**Comment by Skal132** [comment]
###  **Proof of Identity**

Node BlackSkal
5EpxdHj2C18Fi1XC9BcpZYnQTjeuNBCCfcd7JuVQEZHjoKo1

![image](https://user-images.githubusercontent.com/114217272/191915858-6fd8c906-de03-45f5-a481-f24def4e1658.png)


Email address:

![image](https://user-images.githubusercontent.com/114217272/191914614-26ece50f-1c5e-4856-997f-d533e73334fb.png)

Twitter account:
https://twitter.com/skal132

![image](https://user-images.githubusercontent.com/114217272/191915024-772e8061-8a27-4e1c-8842-3fb90d6e2588.png)

**Comment by ipapandinas** [comment]
HERACLES-02 proof of identity

Adress
5DtqncDS1BCTjneX3kskvb8NznQT4KFcYYh9Q6TG7U699R3a

<img width="877" alt="image" src="https://user-images.githubusercontent.com/26460174/192524496-c76e0697-f20b-4832-a28e-79940afc1495.png">

Identity is the same from the previous message https://github.com/capsule-corp-ternoa/ternoa-hub/discussions/5#discussioncomment-3040616 above.

Website : https://heracles.works/

![websiteHeracles](https://user-images.githubusercontent.com/26460174/176201887-be69d117-0312-4ed8-9135-7beac9379301.png)

Twitter : https://twitter.com/heracles_works

<img width="675" alt="twitterHeracles" src="https://user-images.githubusercontent.com/26460174/176202264-569bc5ef-d56d-4f01-831d-e3c6a443d42f.png">

Mail : contact@heracles_works

![mailHeracles](https://user-images.githubusercontent.com/26460174/176202367-3b89452b-56a7-4a7b-9a94-7da70ceb4a97.png)

**Comment by squall-ternoa** [comment]
SQUALL-VALIDATOR - 01 :  Proof of Identity  

Node Address 
5F4SGjxZ8RvJ9NzNixHrAjC67jhd8sby4AzywkobzB5AxDBT

<img width="309" alt="image" src="https://user-images.githubusercontent.com/117892991/201024956-b571bec2-b86e-4a83-9ad9-848e1130eb30.png">

Email :  ternoa@squallnetwork.net

<img width="337" alt="image" src="https://user-images.githubusercontent.com/117892991/201026427-f7227db3-cb7d-4693-a8e1-ac7f946e82f4.png">

**Comment by samgaze** [comment]
Scove-Node-001 - 01 : Proof of Identity

Node Address:
5GmtvQrvi6rpZjGjnhbHtEP28xYjZ9g773CeKg3gpCu5zxAf

![image](https://user-images.githubusercontent.com/14809407/224806248-eb1f28a2-e61a-46ed-9fec-23fdca3ea823.png)

Email: contact@scove.io

![image](https://user-images.githubusercontent.com/14809407/224807007-ee2a2f42-bb6e-4847-8622-172f4defd5ac.png)

Twitter:
![image](https://user-images.githubusercontent.com/14809407/224807709-dbfc0ee2-58a4-4cbc-92ec-a628f1082159.png)

**Comment by Abacus243** [comment]
Proof of identity : Abacus
Address : 5E4ByRGa5s5c8sy1rP6sUUEocCv97ozGCJCcB3RrE25obUzB

![Ternoa Abacus](https://github.com/capsule-corp-ternoa/ternoa-hub/assets/133030500/fa07949f-0868-45f0-8df1-79922d3613c7)

**Comment by squall-ternoa** [comment]
SQUALL-VALIDATOR - 02 : Proof of Identity 

Node Address : 5FLEnqfpr8bojjXNvBnBsHFtUmQZ1ZKf8uY7tmhtP4sN7R9c

<img width="311" alt="image" src="https://github.com/capsule-corp-ternoa/ternoa-hub/assets/117892991/c479ff78-b13b-4e3b-bef6-cf0b27e25b0a">

Email : [ternoa@squallnetwork.net](mailto:ternoa@squallnetwork.net)
